db/migrate/20240509104059_create_api_keys.rb

```diff
@@ -0,0 +1,9 @@
+class CreateApiKeys < ActiveRecord::Migration[7.0]
+  def change
+    create_table :api_keys, id: :uuid do |t|
+      t.timestamps
```

Comment:
If we are going to the trouble of storing this should we add a 'description' of the keys so we can understand what is using the API?